README.md

```diff
@@ -1,3 +1,5 @@
+**NOTE: This repository is dead. See #559 for more information, including a V2.**
```

Comment:
I believe you have to manually link issues in the README unless it's been changed in the past couple weeks. Might be nice to add that link so it's easier to access the issue.